docs/.vuepress/sidebar/en.ts

```diff
@@ -279,6 +279,10 @@ export const enSidebar = sidebar({
               text: "IDE",
               link: "/build/tooling/ide.md"
             },
+            {
+              text: "Sepolia Explorers",
+              link: "/build/tooling/sepolia-explorers.md"
+            }
```

Comment:
We shouldnt have a separate page for Sepolia explorers. It should just be explorers?